src/test/resources/application.yml

```diff
@@ -81,7 +81,10 @@ sandbox:
     xslt-url: https://metis-core-rest-test.eanadev.org/datasets/xslt/default
   enrichment:
     dereference-url: https://metis-dereference-rest-test.eanadev.org
-    enrichment-url: https://metis-enrichment-rest-test.eanadev.org
+    enrichment-properties:
```

Comment:
I am just afraid are this are production credentials? :sweat_smile: if so we need to remove them and use test environments.

Reply:
Added test info